flutter-studio/src/io/flutter/actions/FlutterNewProjectAction.java

```diff
@@ -51,11 +61,28 @@ public void actionPerformed(AnActionEvent e) {
         }
       }
     }
-    NewProjectUtil.createNewProject(getEventProject(e), wizard);
+    createNewProject(getEventProject(e), wizard);
```

Comment:
Perhaps a comment explaining where the the inlined `createNewProject` came from?

Reply:
Done